backon/Cargo.toml

```diff
@@ -20,7 +20,8 @@ targets = [
 ]
 
 [features]
-default = ["tokio-sleep", "gloo-timers-sleep"]
+default = ["std-blocking-sleep", "tokio-sleep", "gloo-timers-sleep"]
+std-blocking-sleep = []
 gloo-timers-sleep = ["dep:gloo-timers", "gloo-timers?/futures"]
 tokio-sleep = ["dep:tokio", "tokio?/time"]
```

Comment:
I may prefer default to none and only opt-in features when necessary.

For example, most users of `tokio-sleep` don't need `gloo-timers-sleep`. Although, currently we can use "default-feature = false".

Reply:
The motivation for this change is to ensure that backon's old users can continue working smoothly after upgrading to v1.0. We can revisit this while preparing for v2.0. However, since this change can break users silently, it might be best to leave it as is.